### ISSUE TITLE
Update records (and keeping notices, tasks and IDs)

### ISSUE THREAD

**Issue by dslengel** [ONLYOFFICE/CommunityServer#295]
Hello dear help team,

The following scenario applies when importing data:
Dublicated entries:
The search for duplicate entries is carried out on the basis of the "E-mail address" field.

1. Skip
2. Overwrite
3. Double up

**but no**
4. Update

Is there a way to update the records? It would be nice to be able to update records without deleting notes and tasks and if possible without changing the IDs.

Thank you for your support,
Dirk

**Comment by Carazyda** [comment]
Hello @dslengel. Did you mean import in CRM module? Describe your case using an example. Why don't you like rewriting? What's different with the update?

**Comment by dslengel** [comment]
Hello Mike,

if I have updated my data elsewhere (this makes sense for certain reasons),
then I would like to import the new information in the master data.
If I do that, all data will be overwritten and all notes and tasks for the
data record will be lost, and the new data record will be given a new ID
and can therefore no longer be accessed via the old link.

To get around that, I have to do the following:
Import the data records as duplicates, mark them, for example, as "import"
by keyword, then call up these "import" data and specify them as duplicates
and combine contacts.
Only then will IDs, notes and tasks be retained. This will take a lot of
time.

Hence the question of whether you could also implement the update option
when importing.


Am Mo., 31. Mai 2021 um 15:15 Uhr schrieb Mike Grunin <
***@***.***>:

**Comment by Carazyda** [comment]
Yes, that's a good suggestion. We will think about it. Added to our private bug tracker under number 50686. Thank you.

**Comment by dslengel** [comment]
Great, thanks!
:-)

Am Mi., 2. Juni 2021 um 11:57 Uhr schrieb Mike Grunin <
***@***.***>:
